src/App.jsx

```diff
@@ -28,8 +28,12 @@ const App = () => {
         let fullActorList = [...actors];
 
         actorList.forEach((actorRow) => {
-          const fullName = [actorRow.first_name, actorRow.last_name];
-          fullActorList.push(fullName);
+          const actor = {
+            firstName: actorRow.first_name,
+            lastName: actorRow.last_name,
+            id: actorRow.id,
+          };
+          fullActorList.push(actor);
```

Comment:
nit: Transforming database naming to naming used by the rest of the codebase is typically best done on the backend before it's exposed to the rest of the application. This helps to make sure any database stuff is encapsulated in the backend so it's easier to change it without affecting the frontend.  
In fact, we should avoid sending any "row" or other stuff direct from the database to the frontend.

Reply:
This might be a job for some middleware that transforms database output into specific js objects, basically the code you have here.

Reply:
Also, this might be a job for array map

Reply:
Excellent tip! Taking care of that processing on the backend makes things a lot clearer on the front end.